CHANGELOG.md

```diff
@@ -11,6 +11,111 @@ git log --first-parent
++ Allow passing smart pointers in place of raw pointers to most matrix functions. This means that things like `vec->compute_norm2(x.get())` or `vec->compute_norm2(lend(x))` can be simplified to `vec->compute_norm2(x)` [#1279](https://github.com/ginkgo-project/ginkgo/pull/1279) [#1261](https://github.com/ginkgo-project/ginkgo/pull/1261)
++ Catch overflows in prefix sum operations, which makes Ginkgo's operations much less likely to crash. This also improves the performance of the prefix sum kernel [#1303](https://github.com/ginkgo-project/ginkgo/pull/1303)
++ Make the installed GinkgoConfig.cmake file relocatable and follow more best practices [#1325](https://github.com/ginkgo-project/ginkgo/pull/1325)
++ Allow using the profiler hooks and inline input strings in benchmarks [#1342](https://github.com/ginkgo-project/ginkgo/pull/1342)
+
+### Fixes
++ Fix the wrong type and pass real-number value with device_type to devices [#1253](https://github.com/ginkgo-project/ginkgo/pull/1253)
++ Fix OpenMPI version check [#1200](https://github.com/ginkgo-project/ginkgo/pull/1200)
++ Fix hwloc related slowdown in MPI applications [#1230](https://github.com/ginkgo-project/ginkgo/pull/1230)
++ Fix hwloc version detection [#1266](https://github.com/ginkgo-project/ginkgo/pull/1266)
++ Fix PAPI detection is non-implicit include directories [#1268](https://github.com/ginkgo-project/ginkgo/pull/1268)
++ Fix PAPI support for newer PAPI versions: [#1321](https://github.com/ginkgo-project/ginkgo/pull/1321)
++ Fix pkg-config file generation for library paths outside prefix [#1271](https://github.com/ginkgo-project/ginkgo/pull/1271)
++ Fix build failures with ROCm 5.4 due to missing Thrust includes [#1214](https://github.com/ginkgo-project/ginkgo/pull/1214)
++ Fix build failures with CUDA 12 due to deprecated functionality [#1235](https://github.com/ginkgo-project/ginkgo/pull/1235)
++ Fix build failures with OneAPI 6 due to changed namespace structure [#1251](https://github.com/ginkgo-project/ginkgo/pull/1251)
++ Fix runtime failures for one-sided MPI wrapper functions observed on some OpenMPI versions [#1249](https://github.com/ginkgo-project/ginkgo/pull/1249)
++ Fix incorrect read for skew-symmetric MatrixMarket files with explicit diagonal entries [#1272](https://github.com/ginkgo-project/ginkgo/pull/1272)
++ Fix handling of missing diagonal entries in symbolic factorizations [#1263](https://github.com/ginkgo-project/ginkgo/pull/1263)
++ Fix segmentation fault in benchmark matrix construction [#1299](https://github.com/ginkgo-project/ginkgo/pull/1299)
++ Fix the mpi cxx type binding by c binding [#1306](https://github.com/ginkgo-project/ginkgo/pull/1306)
++ Fix the additional residual check in IR [#1307](https://github.com/ginkgo-project/ginkgo/pull/1307)
++ Fix the cuSPARSE CSR SpMM issue on single strided vector when SM >= 70 and cuda >= 11.6 [#1322](https://github.com/ginkgo-project/ginkgo/pull/1322)
```

Comment:
```suggestion
+ Fix the cuSPARSE CSR SpMM issue on single strided vector when cuda >= 11.6 [#1322](https://github.com/ginkgo-project/ginkgo/pull/1322) [#1331](https://github.com/ginkgo-project/ginkgo/pull/1331)
```
SM < 70 is failed at cuda >= 11.7 though

Reply:
@tcojean This one still needs to be addressed